script/Deploy.s.sol

```diff
@@ -162,71 +184,133 @@ contract DeployGoerli is GoerliParams, Deploy {
-      chainlinkRelayerFactory.deployChainlinkRelayer(ARB_GOERLI_CHAINLINK_BTC_USD_FEED, ORACLE_INTERVAL_TEST); // live feed
+    // denominations
+    IBaseOracle _arbUSDPriceFeed =
+      denominatedOracleFactory.deployDenominatedOracle(_arbETHPriceFeed, chainlinkEthUSDPriceFeed, false);
 
-    IBaseOracle _stonesWbtcOracle = new OracleForTestnet(0.001e18); // denominated feed 'STN / BTC'
-    IBaseOracle _stonesOracle =
-      denominatedOracleFactory.deployDenominatedOracle(_stonesWbtcOracle, _wbtcUsdOracle, false);
+    IBaseOracle _rethETHOracle = new OracleForTestnet(0.98e18);
+    IBaseOracle _rethOracle =
+      denominatedOracleFactory.deployDenominatedOracle(_rethETHOracle, chainlinkEthUSDPriceFeed, false);
 
-    IBaseOracle _totemWethOracle = new OracleForTestnet(1e18); // hardcoded feed 'TTM'
-    IBaseOracle _totemOracle =
-      denominatedOracleFactory.deployDenominatedOracle(_totemWethOracle, chainlinkEthUSDPriceFeed, false);
+    IBaseOracle _magicETHOracle = new OracleForTestnet(0.00032e18);
+    IBaseOracle _magicOracle =
+      denominatedOracleFactory.deployDenominatedOracle(_magicETHOracle, chainlinkEthUSDPriceFeed, false);
 
-    delayedOracle[WETH] = delayedOracleFactory.deployDelayedOracle(chainlinkEthUSDPriceFeed, ORACLE_INTERVAL_TEST);
-    delayedOracle[FTRG] = delayedOracleFactory.deployDelayedOracle(_opUSDPriceFeed, ORACLE_INTERVAL_TEST);
-    delayedOracle[WBTC] = delayedOracleFactory.deployDelayedOracle(_wbtcUsdOracle, ORACLE_INTERVAL_TEST);
-    delayedOracle[STONES] = delayedOracleFactory.deployDelayedOracle(_stonesOracle, ORACLE_INTERVAL_TEST);
-    delayedOracle[TOTEM] = delayedOracleFactory.deployDelayedOracle(_totemOracle, ORACLE_INTERVAL_TEST);
+    delayedOracle[WSTETH] = delayedOracleFactory.deployDelayedOracle(chainlinkEthUSDPriceFeed, ORACLE_INTERVAL_TEST);
+    delayedOracle[ARB] = delayedOracleFactory.deployDelayedOracle(_arbUSDPriceFeed, ORACLE_INTERVAL_TEST);
+    delayedOracle[CBETH] = delayedOracleFactory.deployDelayedOracle(chainlinkEthUSDPriceFeed, ORACLE_INTERVAL_TEST);
+    delayedOracle[RETH] = delayedOracleFactory.deployDelayedOracle(_rethOracle, ORACLE_INTERVAL_TEST);
+    delayedOracle[MAGIC] = delayedOracleFactory.deployDelayedOracle(_magicOracle, ORACLE_INTERVAL_TEST);
 
     // Setup collateral types
-    collateralTypes.push(WETH);
-    collateralTypes.push(FTRG);
-    collateralTypes.push(WBTC);
-    collateralTypes.push(STONES);
-    collateralTypes.push(TOTEM);
+    collateralTypes.push(WSTETH);
+    collateralTypes.push(ARB);
+    collateralTypes.push(CBETH);
+    collateralTypes.push(RETH);
+    collateralTypes.push(MAGIC);
   }
 
   function setupPostEnvironment() public virtual override updateParams {
     // deploy Camelot liquidity pool to create market price for OD
-    ICamelotV3Factory(GOERLI_CAMELOT_V3_FACTORY).createPool(address(systemCoin), ARB_GOERLI_WETH);
+    ICamelotV3Factory(GOERLI_CAMELOT_V3_FACTORY).createPool(address(systemCoin), address(collateral[WSTETH]));
```

Comment:
Please add a comment. Something like:

// This pool will be against WETH or ETH on mainnet. Using WSTETH on testnet for convenience

We def want the ability to swing the OD market price via this pool, so using our own mintable collateral is very helpful here. I'm glad you made this decision.